docs/source/_templates/algorithm_categories.html

```diff
@@ -0,0 +1,21 @@
+<blockquote> Welcome to the Mantid Algorithm Categories page. This page provides a list of the categories into which the algorithms are split.
```

Comment:
Is there a reason this is a block quote?

It looks inconsistent with other "next level down" pages (e.g. Training and Fitting Overview).